tck-core/src/main/kotlin/io/rsocket/frame/SetupFrame.kt

```diff
@@ -23,14 +29,25 @@ data class SetupFrame(
                 writeShort(it.length.toInt())
                 it.token.preview { this@headerBuffer.writeBytes(this) }
             }
-            writeByte(metadataMimeType.length.toInt())
-            writeUtf8(metadataMimeType.text)
-            writeByte(dataMimeType.length.toInt())
-            writeUtf8(dataMimeType.text)
-            payload.metadata?.length?.let(this::writeLength)
+            if (metadataMimeType != null) {
```

Comment:
As I remember mime types are required
so, need to write at least one byte with 0 value

Reply:
@OlegDokuka I've not found if mime type is required or not in RSocket spec. 
My question is should spec says explicitly is mime type required during connection establishment?